api.py

```diff
@@ -306,3 +306,33 @@ async def get(self, request, *args, **kwargs):
         result = await search_location_by_label(query)
         response = result
         return json(response, status=200)
+
+@ns_loc_func.route('/to-DGGS')
+class to_DGGS(Resource):
+    """Function for finding an array of DGGS cells by a loci uri"""
+
+    @ns.doc('find_dggs_by_loci_uri', params=OrderedDict([
+        ("uri", {"description": "Search DGGS cells by loci uri",
+                    "required": True, "type": "string"}),
+    ]), security=None)
+    async def get(self, request, *args, **kwargs):
+        """Calls DGGS table to query DGGS cells by loci uri"""
+        print(request.args)
+        query = str(next(iter(request.args.getlist('uri'))))
+        result = await find_dggs_by_loci_uri(query)
+        return json(result, status=200)
+
+@ns_loc_func.route('/find-at-DGGS-cell')
+class find_at_DGGS_cell(Resource):
+    """Function for finding an array of Loci-i Features by a DGGS cell ID"""
+
+    @ns.doc('find_at_dggs_cell', params=OrderedDict([
```

Comment:
if we could have the same data structure as mentioned above for this api, that would be good.

```
{ 
   "meta" : {
    ...
    },
   "locations" : {
   ...
    }
}
```